tasty-query/shared/src/main/scala/tastyquery/TypeMaps.scala

```diff
@@ -301,12 +294,12 @@ private[tastyquery] object TypeMaps {
               case Some(sym: ClassTypeParamSymbol) => expandParam(sym, preHi)
               case _                               => tryWiden(tp, preHi)
             forwarded.getOrElse {
-              range(super.derivedSelect(tp, preLo).lowerBound, super.derivedSelect(tp, preHi).upperBound)
+              range(super.derivedSelect(tp, preLo) /*.lowerBound*/, super.derivedSelect(tp, preHi) /*.upperBound*/ )
```

Comment:
these are no longer needed?

Reply:
This is a pretty gray area. I still don't understand what's happening with those `range`s. While the refactoring suggests that a wildcard cannot appear here, I'd rather keep a trace that we *used to* deal with wildcards here, in case it comes to bite back in the future.

That said, I turned that into an explicit textual comment to make it clearer.